flycheck.el

```diff
@@ -521,20 +521,43 @@ enabled.  Changing it will not affect buffers which already have
   :package-version '(flycheck . "0.15")
   :safe #'booleanp)
 
+(defconst flycheck-minimum-level-choices
+  '((const :tag "All locations" nil)
+    (const :tag "Informational messages" info)
+    (const :tag "Warnings" warning)
+    (const :tag "Errors" error)
+    (symbol :tag "Custom error level"))
+  "Choices for minimum error level options.
+
+See `flycheck-navigation-minimum-level' and
+`flycheck-navigation-minimum-level'.")
```

Comment:
I guess this is a copy & paste error, isn't it?  One mention of this variable would suffice, wouldn't it? 😉